op-dispute-mon/mon/claims.go

```diff
@@ -117,7 +117,13 @@ func (c *ClaimMonitor) checkGameClaims(
 			}
 		} else {
 			if clockExpired {
-				c.logger.Warn("Claim unresolved after clock expiration", "game", game.Proxy, "claimContractIndex", claim.ContractIndex)
+				// SAFETY: accumulatedTime must be larger than or equal to maxChessTime since clockExpired
+				overflow := accumulatedTime - maxChessTime
+				if overflow >= 10*time.Minute {
+					c.logger.Error("Claim unresolved 10 minutes after clock expiration", "game", game.Proxy, "claimContractIndex", claim.ContractIndex)
```

Comment:
I think this is still a warn as nothing bad has really happened (game hasn't resolved incorrectly etc).  I also don't think we want to tie this to our alert config too tightly, but agree we need to improve these logs as they're pretty noisy.  Maybe we just give 30s or 1m grace before logging a warning so it doesn't create noise just because the claim resolves a couple of blocks late.  Then maybe add a field to the log that reports the length of time since it could have been reviewed - should make it easy to see which ones are causing the alert then.

Could even update the alert to include a link to search the logs for anything where that label is greater than the alert threshold to make it really easy.

Reply:
Very good idea on the alert link. Will make a pass through all added alerts and add these filter-hydrated links.